src/internal/components/drag-handle/styles.scss

```diff
@@ -16,3 +16,7 @@
 .handle:active {
   cursor: grabbing;
 }
+
+.handle-disabled {
+  cursor: default !important;
```

Comment:
Let's try to avoid `!important` directive. You can achieve the same effect with classes specificity.

Reply:
Hey @E-Kaese , any chance we can avoid the `!important` here?